library/src/main/java/com/applandeo/materialcalendarview/DatePicker.kt

```diff
@@ -0,0 +1,98 @@
+package com.applandeo.materialcalendarview
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.widget.AppCompatButton
+import androidx.core.content.ContextCompat
+import com.applandeo.materialcalendarview.listeners.OnSelectionAbilityListener
+import com.applandeo.materialcalendarview.utils.CalendarProperties
+import com.applandeo.materialcalendarview.utils.getMidnightCalendar
+import com.applandeo.materialcalendarview.utils.isMonthAfter
+import com.applandeo.materialcalendarview.utils.isMonthBefore
+import kotlinx.android.synthetic.main.date_picker_dialog.view.*
+
+/**
+ * This class is responsible for creating DatePicker dialog.
+ *
+ * Created by Mateusz Kornakiewicz on 27.07.2017.
+ */
+
+class DatePicker(
+        private val context: Context,
+        private val calendarProperties: CalendarProperties
+) {
+
+    fun show(): DatePicker {
+        val layoutInflater = LayoutInflater.from(context)
+        val view = layoutInflater.inflate(R.layout.date_picker_dialog, null)
```

Comment:
val view = LayoutInflater.from(this).inflate(R.layout.date_picker_dialog, null, false)